README.md

```diff
@@ -6,73 +6,147 @@ Welcome to the monorepo of the Liferay Frontend Infrastructure team.
-| [`@liferay/jest-junit-reporter`](./projects/npm-tools/packages/jest-junit-reporter)                       | [label: `jest-junit-reporter`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajest-junit-reporter)                       | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=npm-tools%2C+jest-junit-reporter&template=jest-junit-reporter.md)                       |
-| [`@liferay/js-insights`](./projects/npm-tools/packages/js-insights)                                       | [label: `js-insights`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajs-insights)                                       | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=npm-tools%2C+js-insights&template=js-insights.md)                                       |
-| [`@liferay/js-publish`](./projects/npm-tools/packages/js-publish)                                         | [label: `js-publish`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajs-publish)                                         | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=npm-tools%2C+js-publish&template=js-publish.md)                                         |
-| [`@liferay/npm-bundler-preset-liferay-dev`](./projects/npm-tools/packages/npm-bundler-preset-liferay-dev) | [label: `npm-bundler-preset-liferay-dev`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Anpm-bundler-preset-liferay-dev) | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=npm-tools%2C+npm-bundler-preset-liferay-dev&template=npm-bundler-preset-liferay-dev.md) |
-| [`@liferay/npm-scripts`](./projects/npm-tools/packages/npm-scripts)                                       | [label: `npm-scripts`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Anpm-scripts)                                       | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=npm-tools%2C+npm-scripts&template=npm-scripts.md)                                       |
+### Liferay DXP Development Tools
+
+These include the toolkits to deal with themes, JavaScript projects, and the AMD Loader, which is the piece of software that comes with [liferay-portal](https://github.com/liferay/liferay-portal) and orchestrates all JS module loading and wiring.
+
+#### Liferay Themes Toolkit
+
+-   Projects
+    -   [Themes Toolkit v10.x +](./projects/js-themes-toolkit): Develop themes for DXP versions 7.2 and above (uses Gulp 4)
```

Comment:
Should we delete that space before the "+"?

Reply:
I had the same doubt... Yeah, let's delete it. :raised_hands: